File: Python/7py 1/calc_ccc.py
"""
|--------------------------------------------------------------------------
| Char Code Calculation
|--------------------------------------------------------------------------
|
"""

def calc(x):
   return len(list(filter(lambda i: i == '7', list(''.join(list(map(lambda i: str(ord(i)), list(x)))))))) * 6


def calc(x):
   return x.join(str(ord(i)) for i in x).count('7') * 6


def calc(x):
   res = 0
   for i in x:
      if ord(i) % 10 == 7:
         res+=6
   return res


def calc(x):
   return sum(6 for c in ''.join(map(str, map(ord, x))) if c== '7')


print(calc('abcdef'), 6)
print(calc('ifkhchlhfd'), 6) 
print(calc('aaaaaddddr'), 30) 
print(calc('jfmgklf8hglbe'), 6)
print(calc('jaam'), 12)

File: PHP/7php 1/calc_ccc.php
<?php
/*
|--------------------------------------------------------------------------
| Char Code Calculation
|--------------------------------------------------------------------------
|
*/

function calc($s) {
   return count(array_filter(str_split(implode('', array_map(fn($i) => ord($i), str_split($s)))), fn($i) => $i == 7)) * 6;
}


function calc($s) {
   $res = '';
   foreach (str_split($s) as $char){
      $res .= ord($char);
   }
   return substr_count($res, '7') * 6;
}


function calc($s) {
   return substr_count(join(array_map('ord', str_split($s))), '7') * 6;
}


print_r(calc("abcdef") . ' ' . 6 . '
');
print_r(calc("ifkhchlhfd") . ' ' . 6 . '
');
print_r(calc("aaaaaddddr") . ' ' . 30 . '
');